test/ts-scenario/steps/debug.ts

```diff
@@ -4,11 +4,12 @@
 
 'use strict';
 
-import { Constants } from './constants';
+import {Constants} from './constants';
 import * as BaseUtils from './lib/utility/baseUtils';
-import { Given, Then } from 'cucumber';
+import {Given, Then} from 'cucumber';
 
-Given(/^I place a scenario start message (.+?)$/, { timeout: Constants.STEP_SHORT as number }, async (featureType: string) => {
+// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Get rid of async if not needed

Reply:
removed async, not sure if we have to return promise here